getting_started/getting_started.py

```diff
@@ -58,13 +60,13 @@
 SPAM = 1
 
 # %% [markdown]
-# Labeling functions can be used to represent many heuristic and/or noisy strategies for labeling data, often referred to as [weak supervision](#).
+# Labeling functions can be used to represent many heuristic and/or noisy strategies for labeling data, often referred to as [weak supervision](https://hazyresearch.github.io/snorkel/blog/ws_blog_post.html).
 # The basic idea of labeling functions, and other programmatic operators in Snorkel, is to let users inject domain information into machine learning models in higher level, higher bandwidth ways than manually labeling thousands or millions of individual data points.
 # **The key idea is that labeling functions do not need to be perfectly accurate**, and can in fact even be  correlated with each other.
 # Snorkel will automatically estimate their accuracies and correlations in a [provably consistent way](https://papers.nips.cc/paper/6523-data-programming-creating-large-training-sets-quickly), and then reweight and combine their output labels, leading to high-quality training labels.
 
 # %% [markdown]
-# In our text data setting here, labeling functions can be keyword matchers:
+# In our text data setting here, labeling functions use keyword matchers:
```

Comment:
How about putting keyword matchers on the next line (and capitalizing) so it look similar to the other things (like Regular Expressions).